cert-manager/README.md

```diff
@@ -0,0 +1,12 @@
+To bootstrap (per the [cert-manager getting started
+guide](https://cert-manager.readthedocs.io/en/latest/getting-started/install.html#installing-with-regular-manifests)):
```

Comment:
nit: may want to pin `latest` to `release-0.6`?

Reply:
done